.prettierrc.json

```
@@ -0,0 +1,6 @@
+
+{
+    "singleQuote": true,
+    "trailingComma": "none",
+    "endOfLine": "crlf"
+  }
```

```suggestion
}

```

Please end this file with a newline